### DISCUSSION TITLE
5.0.0 Catalina Support

### DISCUSSION THREAD

**Discussion by andynameistaken** [asmvik/yabai#1454] [Q&A] [answered]
Is loosing support for Catalina based on any new features that are OS-dependent?
If not is it somehow possible to upgrade to recent version of `yabai` on it?

**Comment by asmvik** [comment, marked as answer]
I don't actually know, I haven't had the opportunity to test/check for Catalina compatibility for years at this point. I know for a fact that High Sierra and Mojave would no longer work due to APIs that are in use. I expect Catalina to hit Apple's end of support in a couple of months when Ventura is released.
